website/client/components/static/faq.vue

```diff
@@ -4,7 +4,8 @@
       .col-12.col-md-6.offset-md-3
         h1 {{ $t('frequentlyAskedQuestions') }}
         .faq-question(v-for='(heading, index) in headings')
-          h2.accordion(@click='setActivePage(heading)') {{ $t(`faqQuestion${index}`) }}
+          h2.accordion(:ref='uniqueRef(index)')
+            a(v-on:click.stop.prevent="" href="#", :id='heading', @click='setActivePage(heading)') {{ $t(`faqQuestion${index}`) }}
```

Comment:
the `href='#'` results in never having a link with hash which can be copied

Reply:
This is true but it does still allow external pages to target specific answers... I didn't want to force the page to only allow a single answer to be open at once, since that would be losing functionality it already has